packages/manager/apps/dedicated/client/app/account/identity-documents/user-identity-documents.html

```diff
@@ -31,8 +31,14 @@
     <!--My Identity document description-->
     <p
         class="oui-tile__body"
+        data-ng-if="!$ctrl.isIndianSubsidiary"
```

Comment:
From the ticket description, I don't think we should hide this message for Indian customers

Reply:
@JacquesLarique the ticket says that we need to replace the current message with a new one for Indian Customers. So, I dont see any issue with the current implementation.

Reply:
ok i removed the condition